code/game/objects/items/wayfinding.dm

```diff
@@ -155,57 +157,48 @@
 		user.put_in_hands(P)
 		P.owner = user.real_name
 
-/obj/machinery/pinpointer_dispenser/attackby(obj/item/I, mob/user, params)
+/obj/machinery/pinpointer_dispenser/attackby(obj/item/I, mob/living/user, params)
 	if(machine_stat & (BROKEN|NOPOWER))
 		return ..()
 
 	if(istype(I, /obj/item/pinpointer/wayfinding))
 		var/obj/item/pinpointer/wayfinding/WP = I
```

Comment:
You've already basically touched all the code that used this--take this time to give it a better variable name.

Reply:
ok